src/acquisition/covidcast/database.py

```diff
@@ -539,7 +544,13 @@ def update_timeseries_timestamp2(
   def get_covidcast_meta(self):
     """Compute and return metadata on all non-WIP COVIDcast signals."""
 
-    sql = '''
+    meta = []
+
+    sql = 'SELECT `source`, `signal` FROM covidcast GROUP BY `source`, `signal` ORDER BY `source` ASC, `signal` ASC;'
```

Comment:
I think it is more appropriate to filter by `is_wip` in this query

Reply:
good call